### PR TITLE
feat: dev

### PR DESCRIPTION
**Pull request by shahumang19**
**- What I did**

- Base Code Setup
- Created Root and Secondary server Connections
- Created Verb Builders and AtKeys
- Created Encryption Utility and Keys Loading Utility
- Created utility class to read keys from string
- Created AtClient class to communicate with atclient
- Completed PKAM Authentication.
- Created Scan Verb Functionality.

**- How to verify it**

- Tests should pass

@JeremyTubongbanua

### CODE REVIEW

File: src/util/encryptionutil.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+
+class EncryptionUtil:
+    IV = b'\x00' * 16
+
+    @staticmethod
+    def aes_encrypt_from_base64(clear_text, key_base64):
+        # clear_text = clear_text.encode('utf-8')
+        key = base64.b64decode(key_base64)
+        cipher = Cipher(algorithms.AES(key), modes.CTR(EncryptionUtil.IV), backend=default_backend())
```
Comment (JeremyTubongbanua) [inline src/util/encryptionutil.py]: Make IV an argument

File: src/util/encryptionutil.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        padder = padding.PKCS7(algorithms.AES.block_size).padder()
+        padded_plaintext = padder.update(clear_text) + padder.finalize()
+        cipher_text = encryptor.update(padded_plaintext) + encryptor.finalize()
+        return base64.b64encode(cipher_text).decode()
+
+    @staticmethod
+    def aes_decrypt_from_base64(encrypted_text, self_encryption_key):
+        cipher_text = base64.b64decode(encrypted_text)
+        key = base64.b64decode(self_encryption_key)
+        cipher = Cipher(algorithms.AES(key), modes.CTR(EncryptionUtil.IV), backend=default_backend())
```
Comment (JeremyTubongbanua) [inline src/util/encryptionutil.py]: Make IV an argument

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
See requested changes

My only other ask would be to update the README by adding your name, example code snippets, and how somebody would contribute and test their code :)

Awesome work!

File: src/util/encryptionutil.py (hunk 1 above)
Comment (shahumang19) [reply src/util/encryptionutil.py]: Done

**Pull request merged**